tests/integration/test_vectordraw.py

```diff
@@ -571,18 +576,21 @@ def test_select_vector(self, click_target):
         if click_target == "line":
             # Find line and click it
             line_elements = board.find_elements_by_css_selector("line")
-            line = self.find_line(vector["expected_line_position"], line_elements)
-            line.click()
+            elem = self.find_line(vector["expected_line_position"], line_elements)
         elif click_target == "tail":
             # Find tail and click it
             point_elements = board.find_elements_by_css_selector("ellipse")
-            tail = self.find_point(vector["expected_tail_position"], point_elements)
-            tail.click()
+            elem = self.find_point(vector["expected_tail_position"], point_elements)
         else:
             # Find tip and click it
             point_elements = board.find_elements_by_css_selector("ellipse")
-            tip = self.find_point(vector["expected_tip_position"], point_elements)
-            tip.click()
+            elem = self.find_point(vector["expected_tip_position"], point_elements)
+
+        # supposed to hover first, then click, on the vector parts
```

Comment:
Idk if this is true, but in manual / debugger testing, I couldn't get it to trigger by clicking the element without first hovering over it.

Reply:
Hmm.. yep, I have the same issue when running this XBlock in my devstack -- I can't select a line without clicking on it.

It seems like an accessibility problem then, since we can't rely on hover when using the keyboard to navigate.  But I don't know if this XBlock is in active use for any of our clients, so don't think there's budget anywhere to fix it..  Could you update this comment to indicate that there's a possible a11y issue here, so we've flagged it?